core/services/commander/main.py

```diff
@@ -167,11 +166,12 @@ async def do_eeprom_update(i_know_what_i_am_doing: bool = False) -> Any:
 async def reset_settings(i_know_what_i_am_doing: bool = False) -> Any:
     check_what_i_am_doing(i_know_what_i_am_doing)
     # Be sure to not delete bootstrap to avoid going back to factory image
-    bootstrap_config_path = "/root/.config/bootstrap/startup.json"
-    temporary_location = "/tmp/bootstrap_startup.json"
-    shutil.copy2(bootstrap_config_path, temporary_location)
-    delete_everything(Path(appdirs.user_config_dir()))
-    shutil.copy2(temporary_location, bootstrap_config_path)
+    ignore = [
+        Path("/root/.config/ardupilot-manager/firmware/logs"),
```

Comment:
```suggestion
        Path("/root/.config/ardupilot-manager/firmware/logs"),
        Path("/root/.config/ardupilot-manager/firmware/scripts"),
        Path("/root/.config/ardupilot-manager/firmware/terrain"),
        Path("/root/.config/ardupilot-manager/logs"),
```
It may be easier to just ignore the whole firmware folder (plus the (t)logs folder). I'm unsure whether we want to delete the current Navigator firmware and the like? They don't seem like BlueOS settings 🤷‍♂️

Reply:
> keep only the logs

You're currently only keeping bin logs. Tlogs seemingly get generated in the logs folder that's outside the firmware folder (or at least they used to? I have one there anyway).

> scripts can be failing or wrong updated..

I agree, but I feel like that's better suited to a genuine "factory reset" feature of the whole onboard computer, since users with custom scripts probably don't consider them to be _settings_ of BlueOS.

Also, will that break BlueBoat configs? I don't imagine "I reset my BlueOS settings and now my NavLight doesn't work" would make sense to many users, since in principle the autopilot's "internal" configuration and setup are independent of BlueOS's settings.

In my mind, "which flight controller board does BlueOS want to connect to?", and "which MAVLink router is BlueOS using?" are BlueOS settings (which are stored in `ardupilot-manager/settings.json`), but everything else related to the autopilot is not a BlueOS setting.